audinterface/core/feature.py

```diff
@@ -415,6 +417,92 @@ def to_numpy(
         """
         return frame.values.T.reshape(self.num_channels, self.num_features, -1)
 
+    def _assert_shape_3d(
+            self,
+            features: typing.Union[np.ndarray, pd.Series]
+    ):
+        r"""Reshape to [n_channels, n_features, n_frames]."""
+
+        if self.process.process_func_is_mono:
+            features = np.array(features)
+            # when mono processing is turned on
+            # the channel dimension has to be 1
+            # so we would usually omit it,
+            # but since older versions required
+            # a channel dimension we have to
+            # consider two special cases
+            if (features.ndim == 4) and \
+                    (features.shape[1] == 1):
+                # (1, features, frames)
+                # -> (channels, 1, features, frames)
+                # -> (channels, features, frames)
+                features = features.squeeze(axis=1)
+            elif (features.ndim == 3) and \
+                    (self.win_dur is None) and \
+                    (features.shape[1] == 1):
+                # (1, features)
+                # -> (channels, 1, features)
+                # -> (channels, features)
```

Comment:
And here
```
(channels, channels, features)
```
to
```
(channels, features)
```

So maybe change the comment to
```python
# (channels, channels, features)
# -> (channels, features)
```

Reply:
Like before but for the case that the we do not process frames.

Reply:
I changed the comment to:

```
# (channels, 1, features)
# -> (channels, features)
```